build/teamcity/cockroach/nightlies/pebble_nightly_metamorphic_crossversion_impl.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+dir="$(dirname $(dirname $(dirname $(dirname "${0}"))))"
+source "$dir/teamcity-bazel-support.sh"  # For process_test_json
+
+set -euxo pipefail
+
+ARTIFACTS_DIR=$PWD/artifacts/meta
+mkdir -p "${ARTIFACTS_DIR}"
+chmod o+rwx "${ARTIFACTS_DIR}"
+chmod -R o+rwx "/test-bin"
+ls -l "/test-bin"
+GO_TEST_JSON_OUTPUT_FILE=$PWD/artifacts/test.json.txt
+
+echo "TC_SERVER_URL is $TC_SERVER_URL"
+
+bazel build //pkg/cmd/bazci //pkg/cmd/github-post //pkg/cmd/testfilter --config=ci
+
+BAZEL_BIN=$(bazel info bazel-bin --config ci)
+
+# The script accepts the arguments accepted by TestMetaCrossVersion. It should
+# look like:
+#
+#   --version release-21.2,f390aeb3d,f390aeb3d.test --version release-22.1,c5e43d21,c5e43d21.test
+#
+# We need to pass these same arguments to the test invocation. To do that,
+# prefix each argument with `--test_arg `, so that we can instruct bazel
+# to set the arguments appropriately.
+test_args=$(echo $@ | python3 -c "import sys; print(' '.join(['--test_arg=' +word.strip() for word in sys.stdin.read().split(' ')]))")
+
+exit_status=0
+$BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci test \
```

Comment:
`bazci`'s command-line has changed a bit,  but luckily it makes this invocation simpler.

Please rebase and make the following change here:

```
$BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci --process_test_failures -- \
    test @com_github_cockroachdb_pebble//internal/metamorphic/crossversion:crossversion_test
    ...
```